js/components/ChartAccordion.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+
+/* ChartAccordion - This collapsible accordion menu, built on
+ * react-acessible-accordion, lives under the "Hidden Items"
+ * tab. It display manuscript and letter items for the columns
+ * and rows that have been hidden on the scriptchart, and
+ * signals the scriptchart to redisplay them when they are
+ * selected.
+ */
+
+import LetterButton from "./LetterButton";
+import SyriacLetter from "./SyriacLetter";
+import ManuscriptItem from "./ManuscriptItem";
+
+const AccordionStyle = {
+  border: "2px dotted gray"
+};
+
+import {
+  Accordion,
+  AccordionItem,
+  AccordionItemTitle,
+  AccordionItemBody
+} from "react-accessible-accordion";
+
+// XXX Should this file be somewhere else? It's needed to style the accordion.
```

Comment:
This should be fine. If we wanted to bundle it elsewhere we could either have Webpack copy it in the Jekyll assets folder, or just copy it and check it in with git, but it doesn't seem necessary.

Reply:
Cool. It's only really applicable to this component, so I think we can just leave the import here for now.